### PR TITLE
Create homework10.py

### PR DESCRIPTION
**Pull request by eugenesamozdran**

### CODE REVIEW

File: homework/jenya_s/homework10.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import requests
```
Comment (aodarc) [inline homework/jenya_s/homework10.py]: rewrite the homework using OOP :)

File: homework/jenya_s/homework10.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    # Getting rid of '\n'
+    for i in raw_list_of_urls:
+        clear_list_of_urls.append(i.replace('\n',''))
+
+    for url in clear_list_of_urls:
+        r = requests.get(url)
+
+        # checking if URL is an image and its size is less or equal to 5 Mb. If yes - downloading the image
+        if r.headers["Content-Type"] in IMAGE_FORMATS and int(r.headers["Content-Length"]) <= 5242880:
```
Comment (aodarc) [inline homework/jenya_s/homework10.py]: what the magic number is it? `5242880 ` it should be the constant.

File: homework/jenya_s/homework10.py (hunk 1 above)
Comment (eugenesamozdran) [reply homework/jenya_s/homework10.py]: completed, please check

File: homework/jenya_s/homework10.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        self.file_path = file_path
+        self.img_size = img_size
+
+    def url_list(self):
+        with open(self.file_path) as f:
+            # Making a list where each element is a URL-line from opened file.
+            # Each element except the last one contains '\n' in the end.
+
+            raw_list_of_urls = f.readlines()
+            clear_list_of_urls = []
```
Comment (aodarc) [inline homework/jenya_s/homework10.py]:
```
clear_list_of_urls = [i.replace('\n','') for i in f.readlines()]  # use list comprehension
```

**Pull request merged**